models/src/node/mod.rs

```diff
@@ -195,7 +195,7 @@ impl BottlerocketNodeSpec {
 #[derive(Clone, Debug, Default, Serialize, Deserialize, Eq, PartialEq, JsonSchema)]
 pub struct BottlerocketNodeStatus {
     #[validate(regex = "SEMVER_RE")]
-    current_version: String,
+    pub current_version: String,
```

Comment:
I don't want to allow this to be `pub` because we need to ensure that the value stored here is a valid semver string. If it's pub, someone could create a mutable `BottlerocketNodeStatus` and then create an invalid object by disregarding the semver rule.

Can we instead use `current_version().to_string()` and keep this member variable private?

Reply:
You might be able to use `SemVer` as the type or create a type to use instead of `String`, but I agree with keeping it private and using an accessor.